pkg/logs/sds/rules.go

```diff
@@ -0,0 +1,84 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//nolint:revive
+package sds
+
+// RulesConfig as sent by the Remote Configuration.
+// Equivalent of the groups in the UI.
+type RulesConfig struct {
+	ID          string       `json:"id"`
+	Name        string       `json:"name"`
+	Rules       []RuleConfig `json:"rules"`
+	IsEnabled   bool         `json:"is_enabled"`
+	Description string       `json:"description"`
+}
+
+// MatchAction defines what's the action to do when there is a match.
+type MatchAction struct {
+	Type           string `json:"type"`
+	Placeholder    string `json:"placeholder"`
+	Direction      string `json:"direction"`
+	CharacterCount uint32 `json:"character_count"`
+}
+
+// StandardRuleConfig as sent by the Remote Configuration;
+type StandardRuleConfig struct {
+	ID                      string   `json:"id"`
+	Name                    string   `json:"name"`
+	Description             string   `json:"description"`
+	Pattern                 string   `json:"pattern"`
+	Tags                    []string `json:"tags"`
+	Type                    string   `json:"type"`
+	DefaultIncludedKeywords []string `json:"default_included_keywords"`
+}
+
+// StandardRulesConfig contains standard rules.
+type StandardRulesConfig struct {
+	Rules []StandardRuleConfig `json:"rules"`
```

Comment:
```suggestion
	Rules []StandardRuleConfig `json:"standard_rules"`
```
this is maybe not necessary I don't have the full json format to see :)